src/Lumper.UI/Views/Pages/EntityEditor/EntityEditorView.axaml

```diff
@@ -244,10 +244,14 @@
                   </DataTemplate>
                 </ItemsControl.DataTemplates>
               </ItemsControl>
-              <StackPanel Orientation="Horizontal" Spacing="4">
-                <Button Command="{Binding Entity.AddString}">Add KeyValue</Button>
-                <Button Command="{Binding Entity.AddIo}">Add IO</Button>
-              </StackPanel>
+              <DockPanel>
+                <Button Command="{Binding Entity.AddString}" HorizontalAlignment="Left" Margin="4 0 ">Add KeyValue</Button>
+                <Button Command="{Binding Entity.AddIo}" HorizontalAlignment="Left" Margin="4 0">Add IO</Button>
+                <HyperlinkButton NavigateUri="{Binding DocumentationUri}"
+                                 Content="VDC Reference"
+                                 HorizontalAlignment="Right"
+                                 IsVisible="{Binding DocumentationAvailable}"/>

```

Comment:
I think you can avoid having to define `DocumentationAvailable` by using `ObjectConverters.IsNotNull` https://docs.avaloniaui.net/docs/reference/built-in-data-binding-converter, just make DocumentationUri a `string?` then return null for the multi-tab VM.